src/templates/iframe.html

```diff
@@ -0,0 +1,15 @@
+<!-- <%= htmlWebpackPlugin.options.warning %> -->
```

Comment:
@vaevictis and I actually had a conversation about this 8 months ago, and we both found it better if we made this simpler for the scaffold user by getting rid of the webpack plugin (HtmlWebpackPlugin) and simply adding a bare `ticket_sidebar.html` file. Every developer knows `.html`, but how many know HtmlWebpackPlugin?

@vaevictis thoughts?

Reply:
HtmlWebpackPlugin is used to insert dynamic url for garden css components and also will be useful when generate hash to bust cache.